### DISCUSSION TITLE
Focusing on distant objects

### DISCUSSION THREAD

**Discussion by robotastic** [AxisCommunications/discussions#91] [VAPIX]
I am trying to set the Focus parameter for distant objects (2-3 miles) on the Q6215. Can I always use 9999 as the value? Does it changes sometime based on the Iris or some other settings? Does the atmosphere ever impact things?

**Comment by mattias-kindborg-at-work** [comment]
Hi @robotastic. For product related questions you will have better luck using the technical support desk. Please see #13 for more information.

**Comment by whittenator** [comment]
> I am trying to set the Focus parameter for distant objects (2-3 miles) on the Q6215. Can I always use 9999 as the value? Does it changes sometime based on the Iris or some other settings? Does the atmosphere ever impact things?

Were you able to come up with the answer for this or a focus-to-distance lookup?
